### PR TITLE
Feat: plugin for intl

### PR DESCRIPTION
**Pull request by ClarkXia**
New plugin for intl based on `react-intl`.

### CODE REVIEW

File: packages/plugin-intl/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# @ice/plugin-i18n
```
Comment (luhc228) [inline packages/plugin-intl/CHANGELOG.md:1]: > 


```suggestion
# @ice/plugin-intl
```

File: packages/plugin-intl/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# @ice/plugin-intl
+
+`@ice/plugin-intl` is a plugin for use with the ice.js framework. It provides a simple way to add internationalization support to your application.
```
Comment (luhc228) [inline packages/plugin-intl/README.md:3]:
```suggestion
`@ice/plugin-intl` is a ice.js plugin. It provides a simple way to add internationalization support to your application.
```

File: packages/plugin-intl/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+> `@ice/plugin-int` is implemented based on `react-intl`.
```
Comment (luhc228) [inline packages/plugin-intl/README.md:5]:
```suggestion
> `@ice/plugin-int` is based on `react-intl`.
```

File: packages/plugin-intl/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+$ npm i @ice/plugin-intl --save-dev
+```
+
+## Usage
+
+Define the plugin in `ice.config.mts`:
+
+```ts
+import { defineConfig } from '@ice/app';
+import i18n from '@ice/plugin-intl';
```
Comment (luhc228) [inline packages/plugin-intl/README.md:19]:
```suggestion
import intl from '@ice/plugin-intl';
```

File: packages/plugin-intl/README.md (hunk 2 above)
Comment (ClarkXia) [reply packages/plugin-intl/README.md:3]: AI 生成不靠谱呀 = =

File: packages/plugin-intl/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# @ice/plugin-intl
+
+`@ice/plugin-intl` is a ice.js plugin. It provides a simple way to add internationalization support to your application.
+
+> `@ice/plugin-int` is based on `react-intl`.
```
Comment (luhc228) [inline packages/plugin-intl/README.md:5]:
```suggestion
> `@ice/plugin-intl` is based on `react-intl`.
```

**Review by wssgcg1213 — APPROVED** [review]

**Pull request merged**